packages/components/src/components/Link/Link.tsx

```diff
@@ -271,37 +274,35 @@ export const Link: FC<LinkProps> = ({
       flexDirection: 'row',
       alignItems: 'center',
     },
+    testOnly_pressed: testOnlyPressed,
   }
 
-  const getTextStyle = (pressed: boolean): TextStyle => {
-    // TODO: Replace with typography tokens
-    const regularFont: TextStyle = {
-      fontFamily: 'SourceSansPro-Regular',
-      fontSize: 20,
-      lineHeight: 30,
-    }
-    const pressedFont: TextStyle = {
-      fontFamily: 'SourceSansPro-Bold',
-      fontSize: 20,
-      lineHeight: 30,
-    }
+  const textStyle: TextStyle = {
+    color: linkColor,
+    fontFamily: 'SourceSansPro-Regular',
+    fontSize: 20,
+    lineHeight: 30,
+    textDecorationColor: linkColor,
+    textDecorationLine: 'underline',
+  }
 
-    const textStyle: TextStyle = {
-      color: linkColor,
-      textDecorationColor: linkColor,
-      textDecorationLine: 'underline',
+  const getViewStyle = (pressed: boolean): ViewStyle => {
+    const defaultStyle: ViewStyle = {
+      flexDirection: 'row',
     }
 
-    return { ...(pressed ? pressedFont : regularFont), ...textStyle }
+    const backgroundColor = isDarkMode ? Colors.grayDark : Colors.grayLighter
+
+    return pressed ? { backgroundColor, ...defaultStyle } : defaultStyle
   }
 
   return (
     <Pressable {...pressableProps} testID={testID}>
       {({ pressed }: PressableStateCallbackType) => (
-        <>
+        <View style={getViewStyle(pressed)}>
```

Comment:
Fairly inconsequential, but pretty sure this could also just be done against the Pressable itself.

Reply:
Actually, turns out this is consequential--poking around on a "what if" the current state breaks the centering of the text alignment if you make the icon bigger (e.g. width 50, height 50). On main branch the link text remains centered, on your branch it aligns near the top.

So either need to also add alignItems center to defaultStyle or rework slightly to do do the pressed styling on Pressable. 

Kind of on the fence: on the Pressable seems "cleaner" in my mind, but maybe moving the style down to the View (and removing from Pressable) is more defensive coding lest future changes add a View here and break the inheritance of style from Pressable similarly.